teuthology/task/install.py

```diff
@@ -926,6 +929,105 @@ def _upgrade_deb_packages(ctx, config, remote, debs):
         ] + ['%s=%s' % (d, version) for d in debs],
     )
 
+@contextlib.contextmanager
+def rh_install(ctx,config):
+    """
+    Installs rh ceph on all hosts in ctx.
+
+    :param ctx: the argparse.Namespace object
+    :param config: the config dict 
+    """
+    with parallel() as p:
+        for remote in ctx.cluster.remotes.iterkeys():
+            distro = teuthology.get_system_type(remote,distro=True)
+            if distro == 'redhatenterpriseserver':
+                log.info("Installing on RHEL node")
+                p.spawn(rh_install_pkgs, ctx, remote)
+            else:
+                raise RuntimeError ( "Test requires RHEL nodes")
+    try:
+       yield
+    finally:
+        if config.get('skip_uninstall'):
+           log.info("Skip uninstall of Ceph")
```

Comment:
All your logging messages should be e.g. "Skipping uninstall", not "Skip install"